Lombiq.Tests.UI/Exceptions/HtmlValidationAssertionException.cs

```diff
@@ -1,51 +0,0 @@
-using Atata.HtmlValidation;
-using Lombiq.Tests.UI.Services;
-using System;
-
-namespace Lombiq.Tests.UI.Exceptions
-{
-    public class HtmlValidationAssertionException : Exception
-    {
-        public HtmlValidationResult HtmlValidationResult { get; }
-        public Uri Address { get; set; }
-        public string Title { get; set; }
-
-        public HtmlValidationAssertionException(
-            HtmlValidationResult htmlValidationResult,
-            UITestContext context,
-            bool createReportOnFailure,
-            Exception innerException)
-            : base(CreateErrorMessage(context, createReportOnFailure), innerException)
-        {
-            HtmlValidationResult = htmlValidationResult;
-            Address = new Uri(context.Driver.Url);
-            Title = context.Driver.Title;
-        }
-
-        public HtmlValidationAssertionException()
-        {
-        }
-
-        public HtmlValidationAssertionException(string message)
-            : base(message)
-        {
-        }
-
-        public HtmlValidationAssertionException(string message, Exception innerException)
-            : base(message, innerException)
-        {
-        }
-
-        private static string CreateErrorMessage(UITestContext context, bool createReportOnFailure)
-        {
-            var url = context.Driver.Url;
-            var title = context.Driver.Title ?? url;
-
-            var message = $"Asserting the HTML validation result on page {url}({title}) failed.";
-
-            return createReportOnFailure
-                ? message + " Check the HTML validation report in the failure dump for details."
```

Comment:
This useful info is now missing.

Reply:
I don't see how I could maintain such an info yet keep the exception general. These seem like conflicting wants.

Reply:
The "Check the HTML validation report in the failure dump for details." message needs to be included in the exception currently named `HtmlValidationResultException`. I'm not talking about a more general exception.